scilpy/reconst/afd_along_streamlines.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
```

Comment:
I think we removed all shebang (`#!/usr/bin/env python`)
(It is not necessary for module, just for script)